server/config/base.js

```diff
@@ -51,8 +51,7 @@ module.exports = {
             // used by haystack-ui-proxy connector
             // proxy: {
             //     baseUrl: 'https://haystack.example.com',
-            //     cookieName: 'SESSION_COOKIE',
-            //     cookieValue: '' // use environment variable HAYSTACK_PROP_CONNECTORS_TRACES_PROXY_COOKIE__VALUE=your-cookie-value
+            //     cookie: 'YOUR_COOKIE_NAME=YOUR_COOKIE_VALUE'
```

Comment:
Suggestion: Terminate cookie name/value with `;` in docs.   Only really strictly required if multiple `name=value` need to be specified.